### ISSUE TITLE
Page Number Tabs/Links Overlap on Mobile

### ISSUE THREAD

**Issue by ejmg** [mmistakes/minimal-mistakes#721]
Hi there,

It appears that the theme's wrapping behavior causes some minor overlap with the tabs for page navigation. I was wondering if there was a way to increase the space between the tabs vertically for the use cases where it is forced to wrap around?

- [x] I believe this to be a bug with the theme --- not Jekyll, GitHub Pages or one of the bundled plugins.
- [x] I have updated all gems with `bundle update`.
- [x] I have tested locally with `bundle exec jekyll build`.
---

## Environment informations

- **Minimal Mistakes version:**  4.1
- **`github-pages` or `jekyll` gem version:** 3.3.1
- **Operating system:** Ubuntu 14.04
- **Mobile Browsers Checked:** Chrome and Firefox

---

## Expected behavior

Tabs for next page, page 2, page 3, etc should not overlap when forced to wrap on mobile browsers

## Steps to reproduce the behavior

Use Chrome's Mobile Device Toolbar (or similar), shrink down screen size to where the bottom tabs wrap, and they will overlap as a result.

The images below were created using Chrome's device toolbar but mimic the exact behavior I see on my own mobile device using Chrome on Android (LG G4):

![screenshot from 2016-12-19 13 06 46](https://cloud.githubusercontent.com/assets/18218174/21325780/8bad23ee-c5ed-11e6-84c0-3da13a0cd49a.png)
![screenshot from 2016-12-19 13 06 26](https://cloud.githubusercontent.com/assets/18218174/21325781/8bad4ea0-c5ed-11e6-9396-76aa435f53ff.png)

**Comment by mmistakes** [comment]
Think you need to update the theme, this was fixed in #694.

![screen shot 2016-12-19 at 3 35 29 pm](https://cloud.githubusercontent.com/assets/1376749/21328218/da712648-c600-11e6-9475-d4534fe93f01.png)

**Comment by ejmg** [comment]
Okay, so when I execute ``gem list --local``, it shows ``minimal-mistakes-jekyll (4.1.0, 4.0.1)``

How do I make sure that the version being built by jekyll with ``bundle exec jekyll build`` does so using 4.1.0? My gem file specifies the version of jekyll but not for minimal mistakes, should I do similarly? i.e.

```
gem "jekyll", "~> 3.3.0"
gem "minimal-mistakes-jekyll", "~> 4.1.0"
```

I'm going to go ahead and move forward with this and see what happens.

**Comment by mmistakes** [comment]
Check `Gemfile.lock`. Whatever version is there is want will be used when running `bundle exec jekyll`. 

All you should need to update is run `bundle update`. If that doesn't work `bundle update minimal-mistakes-jekyll` should. And if that still doesn't work then you can specify a version in your `Gemfile` like you mentioned above and then update.

**Comment by ejmg** [comment]
Update, after doing all of the above I'm still being given the same overlap. The ``Gemfile.lock`` points to 4.1.0, so I don't know why it would still be doing that.

What might be a way to move forward?

**Comment by mmistakes** [comment]
If it says 4.1.0 there then it should be using that version if you're running `bundle exec jekyll build` or `bundle exec jekyll serve`

Only other thing I can think of is `gem uninstall minimal-mistakes-jekyll` and remove all the older versions. 

You don't happen to have any files in _includes that might be older versions and overriding the ones bundled in the gem do you?

**Comment by ejmg** [comment]
thank you for your patience so far:

Picking up with your last recommendation, I deleted that directory since I had no custom changes to it. That led to me deleting most directories minus ``_config.yaml`` and ``Gemfile`` until I realized that in my original setting up of this website (where I had forked it first and pulled it rather than use the gem method) that **I never had set ``theme: minimal-mistakes-jekyll`` in the ``_config.yaml``.

Fixing that fixed the problem when I regenerated the website. I clearly do not fully understand how jekyll and gem works with that kind of mistake. Either way, I have resolved the issue and your patience led me to that solution. Thank you as always for a great theme and for your help here.
